artifacts/ubuntu/ubuntu_test.go

```diff
@@ -15,7 +15,7 @@ import (
 var _ = Describe("Ubuntu", func() {
 	DescribeTable("Inspect should be able to parse checksum files",
 		func(release, mockFile string, details *api.ArtifactDetails, envVariables map[string]string, metadata *api.Metadata) {
-			c := New(release, envVariables)
+			c := New(release, "x86_64", envVariables)
```

Comment:
the same

Reply:
We don't have constants of other values, why this one should be?